packages/components/Molecules/Card/CardPromo.vue

```diff
@@ -0,0 +1,282 @@
+<template>
+  <rpl-link class="rpl-card-promo" :class="modifiers" :href="link.url">
+    <slot name="image">
+      <rpl-responsive-img class="rpl-card-promo__image" v-bind="computedImg" alt="" :srcSet="[{ size: 'xs', height: 534, width: 764  }, { size: 's', height: 200, width: 764  }, {  size: 'm', height: 232, width: 448 }, {  size: 'l', height: 232, width: 333 }]" />
+    </slot>
+    <div class="rpl-card-promo__content">
+      <div class="rpl-card-promo__meta">
+        <slot name="meta">
+          <div class="rpl-card-promo__tag" v-if="tag" >{{ tag }}</div>
+          <div class="rpl-card-promo__topic" v-if="topic" >{{ topic }}</div>
+          <slot name="status">
+            <div class="rpl-card-promo__status" :class="`rpl-card-promo__status--${this.status.toLowerCase()}`" v-if="status" >
+              <rpl-icon class="rpl-card-promo__status-icon" :symbol="statusIcon.symbol" :color="statusIcon.color" size="s" />
+              <span>{{status}}</span>
+            </div>
+          </slot>
+          <div class="rpl-card-promo__date" v-if="computedDate">{{ computedDate }}</div>
+        </slot>
+      </div>
+      <slot name="content">
+        <h2 class="rpl-card-promo__title" v-if="title">{{ trimmedTitle }}</h2>
+        <p class="rpl-card-promo__summary" v-if="summary" >{{ trimmedSummary }}</p>
+      </slot>
+      <slot name="footer">
+      </slot>
```

Comment:
I like slots but not sure if we've tested having flexible contents that could probably affect the expected component layout?